src/blocked.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2009-2021, Redis Labs Ltd.
```

Comment:
We don't typically add licensing information in headers. Also, You aren't Redis Labs.

Reply:
not funny :(